tooling/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
```

Comment:
The maintenance commands are hidden behind a lib and a feature so it does not interfere with the normal build. It is also exposed as an alias in `.cargo/config` to help prospective contributors.